lennykamande/stackoverflow.github.io
# #11061605 Stackoverflow Lite

## What does this PR do?
Create User Interfaces for the stackoverflow project to get a feel of how the site will look like
### Description of Task to be completed?
Have the following UI available.
  * Login Page
  * Signup Page
  * Home Page (containing Top questions)
  * Profile Page (containing All questions asked and answers given)

### How should this be manually tested?
Make sure that you have installed Google chrome browser

* Clone the repo on to your local machine
* Cd into the folder 
* Right click on the Home page and open it with the chrome browser 

### Any background context you want to provide?
There are user interfaces for the stackoverflow Lite that have not been developed yet.
### What are the relevant pivotal tracker stories?
#159720806
#159713029
#159713034
#159713041
#159713045
